### PR TITLE
#157731617: Relation between WorkoutLog and WorkoutSession

### PR DESCRIPTION
**Pull request by King-Benx**
### What does this PR do?
Creates a relation between the workout log and workout session
#### Description of Task to be completed?
Previously, wger only linked the workout session and workout log based on date and this presented some problems in certain areas since it was not able to follow foreign keys with the ORM is some location.

This PR creates a foreign key of the Workout Log.

#### How should this be manually tested?
1. Follow the readme.rst procedures on how to set up the application locally 
2. Make sure you run the migrations command
3. The database should contain a field for log_id in the workout session

#157731617

### CODE REVIEW

File: wger/core/static/js/wger-core.js [hunk 1]
```diff
@@ -119,6 +119,20 @@ function setProfileField(field, newValue) {
...
+function changeWorkoutType (event) {
+
+  if (event.value === '1') {
+    $('#id_sets').val(4);
+    $('#id_sets_value').html($('#id_sets').val());
+    updateAllExerciseFormset();
+  } else {
+    $('#id_sets').val(1);
+    $('#id_sets_value').html($('#id_sets').val());
+    updateAllExerciseFormset();
```
Comment (dondrzzy) [inline wger/core/static/js/wger-core.js]: Dont repeat yourself here. Move the bottom 2 lines of code in both if-else conditions and place it at the bottom. They will always be called in both cases.

File: wger/core/static/js/wger-core.js [hunk 2]
```diff
@@ -697,4 +716,73 @@ $(document).ready(function () {
       '/' + token;
     window.location.href = targetUrl;
   });
+
```
Comment (dondrzzy) [inline wger/core/static/js/wger-core.js]: These are not your changes. When you git pull --rebase. Endeavour to squash and always have your commit at the top.

**Review by dondrzzy — COMMENTED** [review]
These are not your changes. When you git pull --rebase. Endeavor to squash and always have your commit at the top.

File: wger/manager/views/workout.py [hunk 3]
```diff
@@ -87,6 +87,8 @@ def view(request, pk):
     canonical = workout.canonical_representation
     uid, token = make_token(user)
 
+    # for day in workout.canonical_representation['day_list']:
```
Comment (hariclerry) [inline wger/manager/views/workout.py]: Please remove this commented line of code.

**Review by hariclerry — DISMISSED** [review]

**Review by emugaya — DISMISSED** [review]
Fix your commit message

**Pull request merged**